src/main/scala/com.snowplowanalytics/iglu.schemaddl/migrations/Migration.scala

```diff
@@ -115,9 +118,96 @@ object Migration {
    * @return migration map of each Schema to list of all available migrations
    */
   def buildMigrationMap(schemas: List[IgluSchema]): MigrationMap =
-    buildMigrationMatrix(NonEmptyList.fromListUnsafe(schemas))
-      .map(source => (source.schemas.head.self, buildMigration(source)))
-      .groupBy(_._1)
-      .mapValues(_.map(_._2))
+    buildMigrationMatrix(NonEmptyList.fromListUnsafe(schemas)).right match {
+      case None => Map.empty[SchemaMap, NonEmptyList[Migration]]
+      // TODO Enes:
```

Comment:
I don't think there's a meaningful implementation of `Order[SchemaMap]`. The whole point of `OrderedSchemas` is that we can know proper order only after communicating with Iglu Server.

Reply:
Yep, I think you are right. Then, I will pointed out the reason of this hack in comment, leave it like that.